packages/url/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@rollup/plugin-url",
+  "version": "3.0.0",
```

Comment:
We'll have to bump this to `4.0.0` and note in the PR that this contains a breaking change in the form of a new minimum version of Rollup.